### PR TITLE
Moved CSS to lesson 8

### PR DESCRIPTION
**Pull request by elinapro**

### CODE REVIEW

**Comment by elinapro** [comment]
Lesson 8 CSS work was accidentally merged into Lesson 7

**Review by mannsn — COMMENTED** [review]
Nice looking portfolio, love the colors and the image!

Comments:

Index.html  fix indentation - format by right clicking in vscode and then selecting format document

Agree with AI reviewer - make sure you have units for your properties

For the experience  - you styled the font, but try adding borders to the experience list items 


From AI reviewer:
CSS Syntax Issues: In your index.css file, some properties are missing units. Properties like padding and margin should have units specified (e.g., px, em). This can lead to unexpected behavior or errors in your styling. Make sure to add the appropriate units where necessary.

Structure of the CSS File: While you've added styles for various elements, ensure you are specifically addressing all the required points from the instructions, like styling for 'Experience' list items and 'Connect' links. Some aspects might need more specific class or ID selectors to directly target the elements intended for styling according to the instructions.

HTML File Structure and Link Tag Placement: The <link> tags for the Google Fonts and your CSS stylesheet are correctly placed. However, ensure that the stylesheet <link> tag always comes before font-related links for better performance. This is a good practice but doesn't break rules here. Also, make sure to use lowercase for file paths. Change 'CSS/index.css' to 'css/index.css'.

CSS Generic Fonts: When defining a custom font like 'Playwrite PT Guides', it’s good to provide a list with fallback font families, just in case the Google Font doesn’t load. This ensures better compatibility and a smoother user experience.

HTML Accessibility: While not a specific requirement of the task, ensuring that images have descriptive, informative alt text is important for accessibility reasons. Your current alt text could be a bit more descriptive. Consider describing what you are doing in the photo or the context of the image in your alt tags.

Summary of Good Work:

You have successfully linked your CSS file and Google Fonts, demonstrating understanding of how to incorporate external resources in an HTML document.

You have made a good start by applying various styles and showing creativity with your choice of fonts and colors.

The use of semantic HTML tags and organization of content into sections is clear and logical, enhancing readability and maintainability of your code. Keep up the good work!

File: CSS/index.css [hunk 1]
```diff
@@ -2,15 +2,15 @@ header{
   background-color: azure;
   color:brown
```
Comment (mannsn) [inline CSS/index.css]: Missing semicolon

**Review by mannsn — APPROVED** [review]
You have a few minor things left, but am marking it as approved.   Take a look at the comments and you can either fix here before you merge, or you can fix with the next lesson - your choice!

From AI reviewer :


In your CSS file, a few style rules are missing semicolons. For consistency and error prevention, always add semicolons at the end of each CSS property-value pair. For example, make sure to add a semicolon after color:brown inside the header block. This can sometimes cause unexpected behavior if not corrected.

index.css - 'font-weight' CSS Property:

You used font-weight:900px and font-weight: 400px. Remember that 'font-weight' takes numerical values without units or keywords like 'bold' and 'normal'. Remove the 'px' to ensure the rule applies correctly.

index.css - Image Styles:

While you have set border styles and width, consider using shorthand to combine these border attributes for your img elements, which can make the CSS files easier to read and maintain.

index.css - Section Spacing:

You've successfully added padding and margin, but be mindful of consistency across all sections to match the varying design needs of the page.

Good Efforts
Successfully linked the CSS stylesheet before the closing </head> tag in index.html.
Changed the background color and text color of the body element in the CSS file.
Loaded a font from Google Fonts in index.html and integrated it into your CSS styling.
Implemented padding and margin between different sections on the page.
Correctly transformed the style of the "Experience" list items into styled blocks with distinct fonts.
Incorporated images into your "Connect" links, addressing one of the stretch goals. Keep it up!

**Pull request merged**